Objects/TrackerHolder.cs

```diff
@@ -47,6 +48,16 @@ public static string GetGhostDirectory(LevelData level = null)
         {
             if (level == null)
                 level = Singleton<Game>.Instance.GetCurrentLevel();
+
+            // currentPlatform is set to Platform.BITCODE on Xbox version
+            bool isXbox = GameData.currentPlatform != Platform.STEAM;
```

Comment:
I don't have the game on steam so can't be sure this works but this seems like the most reliable way to tell after looking around a bit